coalib/results/Diff.py

```diff
@@ -100,6 +100,20 @@ def __len__(self):
         return len(self._changes)
 
     @property
+    def changes_size(self):
+        """
+        Returns total number of changed lines, including lines that will be
+        added as changed.
+        """
+        size = 0
+        for line_diff in self._changes.values():
+            if line_diff.change or line_diff.delete:
+                size += 1
+            if line_diff.add_after:
+                size += len(line_diff.add_after)
+        return size
```

Comment:
shouldn't you rather update the `__len__` function to be better? Also you can simplify this: `sum(len(line_diff.add_after for line_diff in self._changes.values() if line_diff.add_after else 1)`